bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicyImpl.java

```diff
@@ -269,9 +269,9 @@ public RackawareEnsemblePlacementPolicyImpl initialize(ClientConfiguration conf,
                 }
             } catch (RuntimeException re) {
                 if (!conf.getEnforceMinNumRacksPerWriteQuorum()) {
-                    LOG.error("Failed to initialize DNS Resolver {}, used default subnet resolver : {}",
+                    LOG.error("Failed to initialize DNS Resolver {}, used default subnet resolver : {} {}",
```

Comment:
i'd just flip the order of `re` and  `re.getMessage()` on the next line and leave an exception without matching `{}` to let logger print the stack. Or remove `re.getMessage` altogether with both `{}` at the end

Reply:
I remeber only one exception args can print without matching `{}`, I remove the re.getMessage.